LICENSE

```diff
@@ -1,203 +0,0 @@
-
```

Comment:
Is it correct to remove this file? I would think it has to stay, ref. todo 3b) in the guide on how to switch to DCO. There it says this file should only be edited, i.e., remove the API and 3rd Party section if it was there.

Reply:
yes